client/src/styles/styleDefaults.css

```diff
@@ -0,0 +1,37 @@
+@import './propertySets.css';
+
+:root {
+    /*  Colours */
+  --appColour: #153c7a;
+  --appColourLight: #6183ba;
+  --appColourLightest: #b5c9e8;
+  --redColourLightest: #f7dfe4;
+  --successColour: #2ecc71;
+  --brighterSuccess: #11e268;
+  --failColour: #ff0000;
+  --appColourDarker: #0c2348;
+  --appColourDarkest: #051022;
+
+    /*  Fonts   */
+  --fontFamily: Helvetica, Arial;
```

Comment:
Is there a reason we're using Helvetica and Arial here without including the `openSans` font we defined below?

Reply:
not really, I just didn't remove the variables when I brought over the styles page.